pylint/config/callback_actions.py

```diff
@@ -38,3 +47,261 @@ def __call__(
         option_string: Optional[str] = None,
     ) -> None:
         return None
+
+
+class _AccessRunObjectAction(_CallbackAction):
+    """Action that has access to the Run object."""
+
+    def __init__(
+        self,
+        option_strings: Sequence[str],
+        dest: str,
+        nargs: None = None,
+        const: None = None,
+        default: None = None,
+        arg_type: None = None,
+        choices: None = None,
+        required: bool = False,
+        help_msg: str = "",
+        metavar: str = "",
+        **kwargs: "Run",
+    ) -> None:
+        self.run = kwargs["Run"]
```

Comment:
This is a trick to be able to pass the `Run` object to these callbacks. At some point I might investigate whether alle these arguments are actually needed.